src/services/routes/index.js

```diff
@@ -22,24 +22,34 @@ router.get("/api/v1/routes", (req, res, next) => {
 router.post("/api/v1/routes", (req, res, next) => {
   getData(`INSERT INTO routes (name, description, author) VALUES ('${req.body.route.name}', '${req.body.route.description}', '${req.body.route.author}') RETURNING *;`)
     .then(routeResults => {
-      if (routeResults.error) {
-        res.status(404).send({ routeError: routeResults.error });
-      } else {
-        getData(`INSERT INTO places (name, description, longitude, latitude, item, route) VALUES ('${req.body.place1.name}', '${req.body.place1.description}', '${req.body.place1.longitude}', '${req.body.place1.latitude}', '${req.body.place1.item}', '${parseInt(routeResults.data[0].id)}') RETURNING *;`)
-          .then(placeResults => placeResults.error
-            ? res.status(404).send({ placeError: placeResults.error })
-            : res.send({ body: placeResults.data })
-          );
-      }
+      getData(`INSERT INTO places (name, description, longitude, latitude, item, route) VALUES ('${req.body.place1.name}', '${req.body.place1.description}', '${req.body.place1.longitude}', '${req.body.place1.latitude}', '${req.body.place1.item}', '${parseInt(routeResults.data[0].id)}') RETURNING *;`)
+        .then(placeResults => placeResults.error
+          ? res.status(404).send({ placeError: placeResults.error })
```

Comment:
You no longer need this ternary expression here - the error is caught in the catch statement you wrote below.